esmvaltool/preprocessor/_regrid.py

```diff
@@ -140,7 +157,20 @@ def regrid(cube, target_grid, scheme):
         of the form 'MxN', which specifies the extent of the cell, longitude by
         latitude (degrees) for a global, regular target grid.
     scheme : str
-        The regridding scheme to perform, see `regrid.HORIZONTAL_SCHEMES`.
+        The regridding scheme to perform, choose from (arg : func)
+        `linear`: Linear(extrapolation_mode=`mask`),
```

Comment:
Note that regridding is done with `esmpy` for irregular grids, so I would just mention the names here and not the corresponding `iris` functions, because that would be misleading.

Reply:
just removed them